site/source/tutorials/005_form_widgets/demo/finished/biz-e-corp/src/widgets/Banner.ts

```diff
@@ -0,0 +1,15 @@
+import { WidgetBase } from '@dojo/widget-core/WidgetBase';
+import { WidgetProperties } from '@dojo/widget-core/interfaces';
+import { v } from '@dojo/widget-core/d';
+
+export default class Banner extends WidgetBase<WidgetProperties> {
+	render() {
+		return v('div', {}, [
+			v('h1', {
+				styles: {
+					'text-align': 'center'
```

Comment:
Yeah, I deliberately tut3 to not use styles as its not to be encouraged. I changed to use a `title` attribute.